docs/specs/app2app.md

```diff
@@ -187,13 +195,20 @@ Assume there are two apps, A and B. App A is holding a valid user session, and a
 
    - Extract `code` from the url and use it to perform the code exchange.
 
-#### iOS SDK
+#### iOS
+
+The following parameters will be added to `authenticate()` method:
+
+- `app2appOptions: App2AppOptions?`
+  - app2app options. If `null`, this app cannot authenticate other apps through app2app.
+  - `App2AppOptions` contains the following fields:
+    - `userAuthenticationFlags: SecAccessControlCreateFlags?`: The authentication the user must perform on authenticating another app through app2app flow. Read [this doc](https://developer.apple.com/documentation/security/secaccesscontrolcreateflags) for possible values. Internally, it was used to set the [access control flags](https://developer.apple.com/documentation/security/1394452-secaccesscontrolcreatewithflags) of the device key.
```

Comment:
We should follow existing options we have for biometric. See BiometricAccessConstraint and BiometricLAPolicy.